src/WMBot/Commands/Channel.cs

```diff
@@ -167,6 +167,7 @@ public static void PartChannel(Channel channel, string user, string host, string
                         lock (Configuration.Channels)
                         {
                             channel.Remove();
+                            IRC.DeliverMessage("Successfully parted " + channel_name + ".");
```

Comment:
The idea of this is the bot sends a message to the channel where the command is used, to provide confirmation to the person doing the command. At the moment, it doesn't say anything, so one has to do the command again so that the bot says it's not in the channel which was parted to know that the channel was parted.